gap/Algebroids.gi

```diff
@@ -66,6 +105,46 @@ InstallMethod( SetOfGeneratingMorphisms,
         
   A -> List( Arrows( UnderlyingQuiver( A ) ), o -> A.(String( o ) ) ) );
 
+##
+InstallMethod( AssignSetOfGeneratingMorphisms,
+        [ IsAlgebroid and HasUnderlyingQuiver, IsString ],
+  function( A, label )
+    local names, morphisms, func;
+    
+    names := List( Arrows( UnderlyingQuiver( A ) ), String );
+    
+    morphisms := SetOfGeneratingMorphisms( A );
+    
+    func := function( name, m )
+              
+              if Int( name ) <> fail and label = "" then
+                  Error( "The second argument should be a non-empty string" );
+              fi;
+              
+              name := Concatenation( label, ReplacedString( name, "-", "m" ) );
+              
+              MakeReadWriteGlobal( name );
+              
+              DeclareSynonym( name, m );
+              
+              return 1;
+              
+            end;
+            
+    ListN( names, morphisms, func );
+    
+end );
+
+##
+InstallOtherMethod( AssignSetOfGeneratingMorphisms,
+        [ IsAlgebroid and HasUnderlyingQuiver ],
+  function( A )
+    
+    AssignSetOfGeneratingMorphisms( A, "" );
```

Comment:
What does this line mean?

Reply:
Same as above, it gives the flexibility to use the names of the arrows or first attach some string to them.